libs/blocks/merch/merch.js

```diff
@@ -179,10 +180,20 @@ export async function fetchEntitlements() {
   return fetchEntitlements.promise;
 }
 
+async function loadDefaultPriceLiterals() {
+  return fetch(DEFAULT_PRICE_LITERALS_PATH);
+}
+
 export async function fetchLiterals(url) {
   fetchLiterals.promise = fetchLiterals.promise ?? new Promise((resolve) => {
     fetch(url)
-      .then((response) => response.json().then(({ data }) => resolve(data)));
+      .catch(() => loadDefaultPriceLiterals())
+      .then(async (response) => {
+        if (response.ok) return response.json();
+        return loadDefaultPriceLiterals()
+          .then((defaultPriceLiterals) => defaultPriceLiterals.json());
+      })
+      .then(({ data }) => resolve(data));
```

Comment:
I would suggest to change the `loadDefaultPriceLiterals` a little to return the data and call it `getDefaultPriceLiterals`
```suggestion
      .then((response) => response.json().then(({ data }) => resolve(data)))
      .catch(() => defaultPriceLiterals())
```

Reply:
This whole code is now removed.